### PR TITLE
Update to Preact X

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
- Remove Preact 8 and preact-compat
- Update Preact to Preact 10
- Remove preact-x alias in webpack

## Why?
- as we only use `preact-compat`, migrating to preact 10 should be simple
- when updating `@guardian/braze-components` we had challenges of braze picking up the wrong preact version for hooks

## CODE smoke test coverage
- [x] Crossword
- [x] Accessibilty
- [x] Identity

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
  */
 
 // @flow
-import React, { Component, render } from 'preact-compat';
```
Comment (QuarpT) [inline static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js:14]: did you mean to change the javascripts.flow.archive/ files?
(I don't know why we have this archive folder)

Comment (gtrufitt) [reply static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js:14]: They're a reference for the types if someone needs to type a file with Typescript (doesn't really matter whether the files are changed or not). 

Side note, maybe we could drop them into another repo as it makes search frustrating.

**Review by buck06191 — APPROVED** [review]
LGTM pending full testing on CODE.

File: static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js (hunk 1 above)
Comment (mxdvl) [reply static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js:14]: files to exclude


I just exclude `*flow.archive/`

**Pull request merged**